tgbotapi.core/src/commonMain/kotlin/dev/inmo/tgbotapi/types/message/payments/PaidMediaToTelegramPaidMedia.kt

```diff
@@ -0,0 +1,27 @@
+package dev.inmo.tgbotapi.types.message.payments
+
+import dev.inmo.tgbotapi.types.media.TelegramPaidMedia
+import dev.inmo.tgbotapi.types.media.TelegramPaidMediaPhoto
+import dev.inmo.tgbotapi.types.media.TelegramPaidMediaVideo
+
+fun PaidMedia.toTelegramPaidMediaOrNull(): TelegramPaidMedia? = when (this) {
+    is PaidMedia.Photo -> TelegramPaidMediaPhoto(
+        file = this.photo.biggest.fileId
+    )
+    is PaidMedia.Video -> TelegramPaidMediaPhoto(
+        file = this.video.fileId
+    )
+    is PaidMedia.Preview, is PaidMedia.Unknown -> null
+}
+
+fun PaidMedia.Video.toTelegramPaidMediaVideo(): TelegramPaidMediaVideo = TelegramPaidMediaVideo(
```

Comment:
Could you check if it is possible to use `video.toTelegramPaidMediaVideo()` here instead of current body (the code after `=`)?